app/components/form/cancel-with-confirm.hbs

```diff
@@ -1,5 +1,6 @@
 <AuButton @skin="secondary" {{on "click" this.cancel}}>{{yield}}</AuButton>
 
+<AuModalContainer />
```

Comment:
Nice catch! Didn't know this was necessary. I think we mostly use the one in application.hbs

Reply:
It was very annoying adding these everywhere as the frontend keeps failing on it when it is not there 😓